<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Forth Game</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../images/icon.jpg</normaloff>../images/icon.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="next_button">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>240</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 9pt &quot;Rockwell Extra Bold&quot;;
</string>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="image_label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>170</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../images/intro.qrc">:/newPrefix/image1.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ok_button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>383</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 9pt &quot;Rockwell Extra Bold&quot;;
</string>
    </property>
    <property name="text">
     <string>To the next chapter â†’
(only one chance, press directly)</string>
    </property>
   </widget>
   <widget class="QLabel" name="riddle_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(100, 100, 150, 0.6);
color: rgb(250, 250, 250);
font: bold 12pt &quot;Cambria Math&quot;;
border: 0px solid gray;



</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="timer_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(100, 100, 150, 0.7);
font: bold 14pt &quot;Rockwell Extra Bold&quot;;
border: 0px solid black; 
color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="bckg_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>631</width>
      <height>431</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../images/intro.qrc">:/newPrefix/ann.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>bckg_label</zorder>
   <zorder>next_button</zorder>
   <zorder>image_label</zorder>
   <zorder>ok_button</zorder>
   <zorder>riddle_label</zorder>
   <zorder>timer_label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../images/intro.qrc"/>
  <include location="intro.qrc"/>
 </resources>
 <connections/>
</ui>
